components/previous-apprentices/previous-apprentices.module.scss

```diff
@@ -1,11 +1,14 @@
 @use '../../styles/colors' as *;
 @use '../../styles/spacing' as *;
 @use '../../styles/fonts' as *;
+@use '../../styles/mixins' as *;
```

Comment:
I see the mixin being included here, but I don't see the `section-padding` mixin being used. Should that be included? It looks likes there are still some misalignments with this section:
<img width="744" alt="Screen Shot 2021-12-08 at 11 11 03 AM" src="https://user-images.githubusercontent.com/565751/145245367-f656e57b-0b71-454f-bde3-ee3c39b73b60.png">